thoth-tech/handbook
# feat: add vale prose linter

# Description

We would like consistent use of tones and languages across our documentation. To automate the linting process and reduce manual effort in code review, we would like to introduce Vale linter to the workflow and integrate it with the CI pipeline. [Vale](https://docs.errata.ai/vale/about/) is a grammar, style, and word usage linter for the English language.

Relates to [Trello card](https://trello.com/c/GjagPSPF/66-implement-vale-prose-lint-for-markdown-documentation)

## Type of change


# How Has This Been Tested?

1. Checkout this branch
2. Follow the [instructions](https://github.com/thoth-tech/handbook/blob/feat%2Fvale-doc-lint/CONTRIBUTING.md#install-linters) to install the new `Vale` linter.
3. Run the following command
    ```shell
    npm run prose:check
    ```
4. We should expect to see something like the following in the terminal window

<img width="1038" alt="Screen Shot 2022-05-07 at 12 08 43 am" src="https://user-images.githubusercontent.com/756722/167149754-4c7cf190-8776-44a8-af2a-9b4b2806cdbf.png">

# Checklist:
